scripts/src/saforcertadmin/push_secrets.py

```diff
@@ -66,7 +83,15 @@ def create_or_update_repo_secrets(repo, secret_name, key_id, encrypted_value):
     if response.status_code != 201 and response.status_code != 204:
         logging.error(f"unexpected response during put request : {response.status_code} : {response.reason}")
         sys.exit(1)
-    #response_json = response.json()
+    try:
+        response_json = response.json()
+        if "message" in response_json:
+            print(f'[ERROR] updating public key: {response_json["message"]}')
```

Comment:
Also improve the error message here as `[ERROR] updating repo secrets: `

Reply:
as above again, updated.